packages/dialog/src/lib/dialog.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import MUIDialog, { DialogProps as MUIDialogProps } from '@mui/material/Dialog';
+import { Box, DialogActions, DialogContent, DialogTitle, Divider, IconButton, Typography } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+
+export interface DialogProps extends MUIDialogProps {
+  open: boolean;
+  onClose: () => void;
+  title: string;
+  description: string;
+  size?: 'sm' | 'md' | 'lg';
+  children?: React.ReactNode;
+  mainAction: React.ReactNode;
+  secondaryAction?: React.ReactNode;
+  headerDivider?: boolean;
+  footerDivider?: boolean;
+}
+
+export const Dialog = (props: DialogProps) => {
+  const {
+    open,
+    onClose,
+    children,
+    title,
+    description,
+    size,
+    mainAction,
+    secondaryAction,
+    headerDivider,
+    footerDivider,
+    ...other
+  } = props;
+
+  const sizes = {
+    sm: '600px',
+    md: '960px',
+    lg: '1200px',
+    xl: '1920px',
+  };
+
+  const width = sizes[size || 'md'];
+
+  return (
+    <MUIDialog open={open} {...other} PaperProps={{ style: { minWidth: width } }}>
```

Comment:
since you are not using open, you can remove it from the destructuring on line 21 and just pass as "other" here.

removing the logic for "size" and we can just pass 
<MUIDialog  {...other} />

Reply:
Done